src/redis/commands.cr

```diff
@@ -131,7 +131,7 @@ class Redis
     # redis.del("some", "keys", "to", "delete")
     # ```
     def del(*keys)
-      integer_command(concat(["DEL"], keys))
+      del keys.to_a
     end
 
```

Comment:
I'm getting a seg fault after this commit / release of crystal-redis... https://travis-ci.org/aca-labs/crystal-engine-driver/builds/583707080 (linux)
Infinite recursion on OSX

Would it not be easier to call `.to_a.flatten` instead of having the overloaded method (it seems safer)

I haven't found where in my code the issue lies but I assume this is it, as there is no overloaded method for `del` so it looks like it would just recursively call this forever

Reply:
hmm so delete is there. Possibly related to using symbols maybe?
i.e. `del :some, :key` would probably compile and infinitely recurse

Reply:
>  there is no overloaded method for del so it looks like it would just recursively call this forever

Ah... I will fix that, I see.